src/components/templates/ProductTemplate/ProductTemplate/ProductTemplate.tsx

```diff
@@ -37,16 +52,20 @@ function ProductTemplate({
   return (
     <FlexContainer data-automation="ZA.ProductTemplate" className={containerClassnames} gutter={0}>
       <FlexRow gutter={0}>
-        <FlexCol>
-          {showHeader ? (
+        {showHeader ? (
+          <FlexCol>
             <ProductTemplateHeader prevStep={prevStep} progress={progress} onBackPressed={onBackPressed} />
```

Comment:
`onBackPressed` isn't needed as `prevStep` can be a ReactElement meaning you can create your own component with onClick handler